### PR TITLE
Link to .NET Upgrade Assistant from MAUI migration doc

### PR DESCRIPTION
**Pull request by danroth27**

### CODE REVIEW

**Comment by davidbritch** [comment]
@danroth27 I'm against merging this PR. Linking to a marketing doc from a technical doc is never a good idea. I get that the current migration doc isn't helpful, but it was never intended to be the final doc. It's literally what was published last summer, and will be replaced for GA.

**Comment by danroth27** [comment]
> @danroth27 I'm against merging this PR. Linking to a marketing doc from a technical doc is never a good idea. I get that the current migration doc isn't helpful, but it was never intended to be the final doc. It's literally what was published last summer, and will be replaced for GA.

The purpose of adding the link is to point users to instructions on how to actual run the .NET Upgrade Assistant as directed. We could point to https://docs.microsoft.com/dotnet/core/porting/upgrade-assistant-overview instead.

File: docs/get-started/migrate.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ You don't need to rewrite your Xamarin.Forms apps to move them to .NET Multi-pla
 
 The process for migrating a Xamarin.Forms app to .NET MAUI is expected to be:
 
-1. Use the .NET upgrade assistant for .NET MAUI to migrate your Xamarin.Forms projects to .NET MAUI single project, and perform well-known code namespace changes.
+1. Use the [.NET Upgrade Assistant](https://dotnet.microsoft.com/platform/upgrade-assistant) for .NET MAUI to migrate your Xamarin.Forms projects to .NET MAUI single project, and perform well-known code namespace changes.
```
Comment (danroth27) [inline docs/get-started/migrate.md]:
```suggestion
1. Use the [.NET Upgrade Assistant](https://docs.microsoft.com/dotnet/core/porting/upgrade-assistant-overview) for .NET MAUI to migrate your Xamarin.Forms projects to .NET MAUI single project, and perform well-known code namespace changes.
```

**Review by davidbritch — CHANGES_REQUESTED** [review]
You need to drop https://docs.microsoft.com from the link. So the link will be: /dotnet/core/porting/upgrade-assistant-overview

File: docs/get-started/migrate.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ You don't need to rewrite your Xamarin.Forms apps to move them to .NET Multi-pla
 
 The process for migrating a Xamarin.Forms app to .NET MAUI is expected to be:
 
-1. Use the .NET upgrade assistant for .NET MAUI to migrate your Xamarin.Forms projects to .NET MAUI single project, and perform well-known code namespace changes.
+1. Use the [.NET Upgrade Assistant](https://docs.microsoft.com/dotnet/core/porting/upgrade-assistant-overview) for .NET MAUI to migrate your Xamarin.Forms projects to .NET MAUI single project, and perform well-known code namespace changes.
```
Comment (danroth27) [inline docs/get-started/migrate.md]:
```suggestion
1. Use the [.NET Upgrade Assistant](/dotnet/core/porting/upgrade-assistant-overview) for .NET MAUI to migrate your Xamarin.Forms projects to .NET MAUI single project, and perform well-known code namespace changes.
```

**Review by davidbritch — APPROVED** [review]
Thanks @danroth27

**Pull request merged**